packages/blitz/src/cli/index.ts

```diff
@@ -167,8 +166,14 @@ async function main() {
         $ blitz <command>
   
       Available commands
-        ${Object.keys(commands).join(", ")}
-  
+        dev, d          Start a development server 🪄
+        build, b        Create a production build 🏗️
+        start, s        Start the production server 🐎
+        new, n          Create a new Blitz project ✨
+        generate, g     Generate new files for your Blitz project 🤠
+        codegen         Manually trigger the blitz codegen 🤖
```

Comment:
Commenting `generate` as we don't have it working yet
```suggestion
        // generate, g     Generate new files for your Blitz project 🤠
        codegen         Run the blitz codegen 🤖
```

Reply:
not sure that the cli would understand that you don't want to show that to the user hahaha, i'll push a change that deletes it,

Reply:
Pushed